tcms/tests/factories.py

```diff
@@ -317,3 +318,14 @@ class Meta:
 
     run = factory.SubFactory(TestRunFactory)
     user = factory.SubFactory(UserFactory)
+
+
+class SiteFactory(DjangoModelFactory):
+
+    class Meta:
+        model = models.Site
+        django_get_or_create = ('id', )
+
+    id = settings.SITE_ID
+    domain = 'http://test.bogus.nonexistent'
+    name = 'Test site name'
```

Comment:
You do not need a factory here. This record is automatically created in the DB when migrations are applied. The test needs to make sure that the values can be altered as expected.

Reply:
This factory returns current values from the database. I use it in both tests to check what the values are. Can I access the model directly in the test methods without breaking Kiwi's test code consistency?

Reply:
Give it a try and you will see.